net/sshuttle/Makefile

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (C) 2022 Thomas Cujé
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=sshuttle
+PKG_VERSION:=1.1.1
+PKG_RELEASE:=1
+
+PYPI_NAME:=$(PKG_NAME)
+PKG_HASH:=f5a3ed1e5ab1213c7a6df860af41f1a903ab2cafbfef71f371acdcff21e69ee6
+
+PKG_MAINTAINER:=Thomas Cujé <dev.cuje@freenet.de>
+PKG_LICENSE:=LGPL-2.1-only
+
+include ../../lang/python/pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/python/python3-package.mk
+
+define Package/sshuttle
+  SUBMENU:=SSH
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=sshuttle: where transparent proxy meets VPN meets ssh
+  URL:=https://github.com/sshuttle/sshuttle
+  DEPENDS:= \
+  +python3-light \
+  +python3-ctypes \
+  +python3-urllib \
+  +iptables-mod-extra \
+  +iptables-mod-nat-extra \
+  +iptables-mod-ipopt
```

Comment:
Since OpenWrt 22.03 moved from iptables to nftables, thus could be somehow confusing. I think there are two options:
1. Make and do nftables variant only
2. Do it as e.g. miniupnpnd and do two variants one for iptables and second one for nftables

Reply:
Good point! I'll check this.

- [ ] is sshuttle compatible with nftables (seems so at least with NAT mode)?
- [ ] can we specify a list of compatible OpenWRT versions for the package?
- [ ] can we determine the correct package variant with the depencies and what is installed?

Reply:
Sshuttle works perfectly fine with nftables. I removed the dependencies. On OpenWRT 21.03 installations with iptables they might be mandatory though and must be installed manually. AFAIK the DEPENDS keyword does not provide such version checks.